src/index.js

```diff
@@ -0,0 +1,37 @@
+import renderTodo from './modules/renderTodo';
+import setMoreOption from './modules/setMoreOption';
+import './styles.css';
+
+const TodoContainer = document.querySelector('.todo-list');
+const Input = document.querySelector('.add');
+const Form = document.querySelector('form');
+
+let TodoList = [];
+function getTodos() {
+  if (localStorage.getItem('todos')) {
+    TodoList = JSON.parse(localStorage.getItem('todos'));
+  }
+}
+
+getTodos();
+
+function addTodo() {
+  Form.addEventListener('submit', (e) => {
+    e.preventDefault();
+    getTodos();
+    TodoList.push({
+      discription: Input.value,
+      index: TodoList.length !== 0 ? TodoList[TodoList.length - 1].index + 1 : 1,
+      completed: false,
+    });
+    localStorage.setItem('todos', JSON.stringify(TodoList));
+    getTodos();
+    renderTodo(TodoList, TodoContainer);
+    setMoreOption(TodoList, renderTodo, TodoContainer);
+    Input.value = '';
+  });
+}
```

Comment:
- [ ] Great job and kudos to you by doing a great job on this milestone ,you implemented alot functionality on the app as required but still need to work more on the `clear all completed` button according to the project requirement which says Implement a function for the "Clear all completed" button (use filter() method).the clear all completed button not functioning and when over on the button i can see it not working kindly fix this for proper app interractive and user interface     
- [ ] see screenrec below

https://github.com/EngruugbeJimmy/to_do-list-s/assets/92458236/89b80da2-7ab7-441b-b998-858550cf46f7